Formula/postgresql@11.rb

```diff
@@ -123,10 +123,10 @@ def caveats
   end
 
   service do
-    run [opt_bin/"postgres", "-D", var/"postgresql@11"]
+    run [opt_bin/"postgres", "-D", postgresql_datadir]
     keep_alive true
-    log_path var/"log/postgresql@11.log"
-    error_log_path var/"log/postgresql@11.log"
+    log_path postgresql_log_path
+    error_log_path postgresql_log_path
```

Comment:
This may break but if it doesn't: it's nicer than hardcoding these and `postgresql_log_path` isn't used otherwise.